### DISCUSSION TITLE
Prettier?

### DISCUSSION THREAD

**Discussion by ace-beattie** [ciderapp/Cider#995] [General]
Any reason why Cider's repo doesn't use a code formatter like Prettier? Could be useful in standardizing things like tab width (which I've noticed is different in ciderapp/Cider for different files) etc. I totally understand if there's a reason for this, just curious!

**Comment by cryptofyre** [comment]
Linter’s break code a LOT. We’ve used prettier before but its just a hassle to maintain and there’s so many PR’s it wouldn’t make sense to make them follow a single code format.

**Reply by ace-beattie** [reply]
Ahhh got it. Thanks for the quick response :)
